### DISCUSSION TITLE
How to view actual storage usage because of thin provisioning?

### DISCUSSION THREAD

**Discussion by patrickdung** [longhorn/longhorn#5694] [General]
The PVC is thin provisioned in Longhorn.
How do view the total storage space left (combining all nodes) using CLI without using the UI/dashboard?

Thanks.

**Comment by mantissahz** [comment]
It should be `sum(enabledNodes.enabledDisks.AvailableStorage)`
You cloud get values of `AvailableStorage` of nodes from this command `kubectl -n longhorn-system get lhn -o jsonpath='{range.items[*]}{.status.diskStatus.*.storageAvailable}{"\n"}{end}'`

**Reply by patrickdung** [reply]
@mantissahz Thanks, it works.
